nasl/nasl_builtin_find_service.c

```diff
@@ -1879,6 +1911,13 @@ plugin_do_run (struct script_infos *desc, GSList *h, int test_ssl)
                   else if (strncmp (line, "421", 3) == 0
                            && strstr (line, "smtp ") != NULL)
                     mark_smtp_server (desc, port, origline, trp);
+                  else if (line[0] != '\0' && (strstr (buffer, "mysql") != NULL)
+                           && (regex_match (buffer,
+                                            "[0-9a-z]+@[0-9a-z]+ release")
```

Comment:
```suggestion
                                            "[0-9.]+ [0-9a-z]+@[0-9a-z]+ release")
```

to make this regex a little bit more strict and to avoid false detection.

Reply:
Oh! I forgot that part. Thanks @cfi-gb